src/main/kotlin/com/example/fhirvalidator/configuration/ValidationConfiguration.kt

```diff
@@ -49,63 +49,66 @@ class ValidationConfiguration(private val implementationGuideParser: Implementat
     @Bean
     fun validationSupportChain(
         fhirContext: FhirContext,
-        hybridTerminologyValidationSupport: HybridTerminologyValidationSupport,
+        optionalRemoteTerminologySupport: Optional<RemoteTerminologyServiceValidationSupport>,
         npmPackages: List<NpmPackage>
     ): ValidationSupportChain {
         val supportChain = ValidationSupportChain(
             DefaultProfileValidationSupport(fhirContext),
+            SnapshotGeneratingValidationSupport(fhirContext),
             CommonCodeSystemsTerminologyService(fhirContext),
-            hybridTerminologyValidationSupport,
-            SnapshotGeneratingValidationSupport(fhirContext)
+            InMemoryTerminologyServerValidationSupport(fhirContext)
         )
+
         npmPackages.map(implementationGuideParser::createPrePopulatedValidationSupport)
             .forEach(supportChain::addValidationSupport)
 
+        if (optionalRemoteTerminologySupport.isPresent) {
+            val remoteTerminologySupport = optionalRemoteTerminologySupport.get()
+            //val cachingRemoteTerminologySupport = CachingValidationSupport(remoteTerminologySupport)
```

Comment:
Happy to re-enable caching here as we're not going to be using remote terminology validation in production